cmd/operator-sdk/cli/cli.go

```diff
@@ -42,91 +30,71 @@ import (
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"sigs.k8s.io/kubebuilder/pkg/cli"
+	kbgov1 "sigs.k8s.io/kubebuilder/pkg/plugin/v1"
+	kbgov2 "sigs.k8s.io/kubebuilder/pkg/plugin/v2"
```

Comment:
Hi @estroz, @hasbro17 

The Project file scaffolded from here is: 

```
$ cat PROJECT 
domain: my.domain
layout: go.kubebuilder.io/v2.0.0
repo: test
version: 3-alpha

``` 

-  Should not the layout be `go.operator-sdk.io/v3-alpha`?

Reply:
We're using kubebuilder's go plugin, which sets the layout.

Note: plugin version (`v2.0.0`) is different from project version (`v3-alpha`).

Reply:
How we will know if the project was scaffolded by sdk or kubebuilder tool? 
I thought that this info would be in the layout.

Reply:
Since we're eventually going to want Makefile targets for generating and pushing bundles, deploying the operator etc, we may have to wrap the base kubebuilder plugin in our own. Then `layout` will be something like `go.operator-sdk.io/v2.0.0`. Out of scope for this PR though.

Reply:
Solved